hyperapp.d.ts

```diff
@@ -0,0 +1,45 @@
+export as namespace hyperapp
+
+// VNode part
+
+export interface VNode<Data> {
```

Comment:
Hmm, so this is only useful if the user defines an interface for the VirtualNode data? Hmm. Nice.